.env.appStore.example

```diff
@@ -108,6 +108,11 @@ VITAL_REGION="us"
 # @see https://github.com/calcom/cal.com/blob/main/packages/app-store/zapier/README.md
 ZAPIER_INVITE_LINK=""
 
+#  - MAKE
+# Used for the Make integration
+# @see https://github.com/calcom/cal.com/blob/main/packages/app-store/make/README.md
```

Comment:
Didn't see any instructions on how to get the actual MAKE_INVITE_LINK

Reply:
here is the invite link to test: [make.com/en/hq/app-invitation/6cb2772b61966508dd8f414ba3b44510](https://www.make.com/en/hq/app-invitation/6cb2772b61966508dd8f414ba3b44510)

We need to add invite link in general to the setup page as long this app is not yet public, left a comment here: https://github.com/calcom/cal.com/pull/8897#discussion_r1304787291

Reply:
We don't need to add env variables for apps now. So removing it. We do need the instruction on where to get the invite link though.

Reply:
We're using the invite link in Carina's comment above. I'll open another PR to push the app for review on Make so we have a public app. Until then we can use this invite link: [make.com/en/hq/app-invitation/6cb2772b61966508dd8f414ba3b44510](https://www.make.com/en/hq/app-invitation/6cb2772b61966508dd8f414ba3b44510)

Pls lmk if any edits are needed

Reply:
Yeah. We just need to add that link at admin/apps locally.
<img width="1310" alt="Screenshot 2023-08-31 at 12 42 33 PM" src="https://github.com/calcom/cal.com/assets/1780212/cf05b7db-e66f-4d4c-b35b-4baa9d199f36">